server/src/workspace/workspace-sync-metadata/standard-objects/person.object-metadata.ts

```diff
@@ -24,7 +29,7 @@ export class PersonObjectMetadata extends BaseObjectMetadata {
     icon: 'IconUser',
   })
   @IsNullable()
-  name: string;
+  name: object;
```

Comment:
can we also improve this typing? Object ==> FullName { firstName: string, lastName: string }

Reply:
Yes I'm planning to add new types for composite fields and wanted to clarify that those were objects in this PR for now @charlesBochet

Reply:
So you want to do that in another PR?

Reply:
@charlesBochet no I'm adding this now 👍

Reply:
Done @charlesBochet